src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -1017,16 +1017,15 @@ public function testColumnStyle()
 
     public function testThrowsWhenTheCellInAnArray()
     {
-        $table = new Table($this->getOutputStream());
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('A cell must be a TableCell, a scalar or an object implementing "__toString()", "array" given.');
+        $table = new Table($output = $this->getOutputStream());
```

Comment:
this line should stay in the initialization IMO.

Reply:
reverted